torch/linalg/__init__.py

```diff
@@ -88,6 +88,43 @@
 Alias of :func:`torch.det`.
 """)
 
+slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
+linalg.slogdet(input) -> (Tensor, Tensor)
+
```

Comment:
"of the absolute values of the determinants of the determinant(s) of a square matrix or batches of square matrices :attr:`input`." -> "of the absolute value of a square matrix's determinant, or of the absolute values of the determinants of a batch of square matrices :attr\`input\`."